coderedcms/models/page_models.py

```diff
@@ -845,14 +847,20 @@ def convert_to_ical_format(self, dt_start=None, dt_end=None, occurrence=None):
         ical_event.add('summary', self.title)
         # needs to get full page url, not just slug
         desc_str = _('Details')
+        ical_event.add('dtstamp', datetime.now())
```

Comment:
Should this be `timezone.now()` instead? Not sure what dtstamp should contain, if it needs a naive or a tz-aware time.